assets/pages/bundles-landing/support-landing-ab-test/components/contributeNewDesign.jsx

```diff
@@ -3,23 +3,224 @@
 // ----- Imports ----- //
 
 import React from 'react';
+import { connect } from 'react-redux';
 
 import InfoSection from 'components/infoSection/infoSection';
+import InlinePaymentLogos from 'components/inlinePaymentLogos/inlinePaymentLogos';
+import Secure from 'components/secure/secure';
+import CtaLink from 'components/ctaLink/ctaLink';
+import PayPalContributionButton
+  from 'components/payPalContributionButton/payPalContributionButton';
+import { contribCamelCase } from 'helpers/contributions';
+
+import type {
+  Amount,
+  Contrib as ContributionType,
+  ContribError as ContributionError,
+} from 'helpers/contributions';
+import type { Currency } from 'helpers/internationalisation/currency';
+import type { IsoCountry } from 'helpers/internationalisation/country';
+import type { Participations } from 'helpers/abtest';
+import type { ReferrerAcquisitionData } from 'helpers/tracking/acquisitions';
+
+import ContributionSelection from './contributionSelectionNewDesign';
+import GraveError from './graveErrorNewDesign';
+import {
+  changeContribType,
+  changeContribAmountAnnual,
+  changeContribAmountMonthly,
+  changeContribAmountOneOff,
+  changeContribAmount,
+  setPayPalError,
+} from '../../bundlesLandingActions';
+import {
+  getCardLink,
+  getCardCtaText,
+  getCardA11yText,
+} from '../helpers/contributionLinks';
+
+
+// ----- Props ----- //
+
+// Disabling the linter here because it's just buggy...
+/* eslint-disable react/no-unused-prop-types */
+
+type PropTypes = {
```

Comment:
Is this type copy/paste from another place? If that is the case, is it possible to exported there and imported here?

Reply:
Nope, all written fresh 🍅